tests/api/test_categories.py

```diff
@@ -86,14 +87,16 @@ def test_get_category_doesnt_exist(self):
     self.assertEqual(json_response["text"], "Category does not exist")
 
   @require_auth0_secrets()
```

Comment:
Do these tests still require auth0 secrets? If not we should remove these lines.

Reply:
Good catch! Updated :)